content/posts/codegen.md

```diff
@@ -0,0 +1,143 @@
+---
+title: Generating Code
+date: 2021-09-30
+draft: false
+summary: Let's talk about code generation!
+---
+
+# Generating Code
+
+At my day job at Google, I spend a lot of time generating code. Coming from
+outside of Google, this wasn't something I had ever thought about doing.
+Computers writing code for other computers to run has a very Skynet feel to it
+and sounded too complicated to ever be worth the effort.
+Yet, this practice is so commonplace that there's [built-in
+support](https://docs.bazel.build/versions/main/be/general.html#genrule) for it
```

Comment:
CSS nit: you should use the `a:hover` selector so that the UX for links is better. I almost didn't think that this was a hyperlink.